### ISSUE TITLE
PHP exceptions on the php-8.0 container

### ISSUE THREAD

**Issue by markmetcalfe** [totara/totara-docker-dev#183]
When visiting pages (such as any admin page) with php-8.0, we get the following exception:
```
Exception - Object of class admin_category could not be converted to string
```
This seems to only happen on the php-8.0 container. Interestingly, the php-8.0-debug container doesn't experience this issue. There might be something missing in the php-8.0 dockerfile or something.

Full stack trace:
```
line 227 of /lib/classes/collator.php: Error thrown
line 227 of /lib/classes/collator.php: call to Collator->asort()
line 305 of /lib/classes/collator.php: call to core_collator::asort()
line 1285 of /lib/adminlib.php: call to core_collator::asort_objects_by_property()
line 836 of /lib/adminlib.php: call to admin_category->get_children()
line 4095 of /lib/navigationlib.php: call to admin_category->__get()
line 4048 of /lib/navigationlib.php: call to settings_navigation->load_administration_settings()
line 3900 of /lib/navigationlib.php: call to settings_navigation->load_administration_settings()
line 3838 of /lib/navigationlib.php: call to settings_navigation->legacy_initialise()
line 758 of /lib/pagelib.php: call to settings_navigation->initialise()
line 815 of /lib/pagelib.php: call to moodle_page->magic_get_settingsnav()
line 8084 of /lib/adminlib.php: call to moodle_page->__get()
line 149 of /course/management.php: call to admin_externalpage_setup()
```

**Comment by derschatta** [comment]
maybe this is related to the opcache or new JIT compiler

**Comment by derschatta** [comment]
will be solved for now with #184
